### PR TITLE
Make calendar cloud platform neutral

### PR DESCRIPTION
**Pull request by Bugsbane**
I've tried to take out references to any cloud platform, although there are a couple of places where that didn't quite seem possible, so I've made those references to "ownCloud/Nextcloud". Links to docs etc have been kept the same.

### CODE REVIEW

**Comment by DeepDiver1975** [comment]
@Bugsbane there are a lot of files like .calcontroller.js.kate-swp in this pr - please remove them - THX

**Comment by Bugsbane** [comment]
Sorry about that. ...still learning! So does this look better?

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+# Calendar 
 
 [![Build Status](https://travis-ci.org/owncloud/calendar.svg?branch=master)](https://travis-ci.org/owncloud/calendar)
 [![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/owncloud/calendar/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar/?branch=master)
 [![Code Coverage](https://scrutinizer-ci.com/g/owncloud/calendar/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar/?branch=master)
 [![Coverage Status](https://coveralls.io/repos/github/owncloud/calendar/badge.svg?branch=master)](https://coveralls.io/github/owncloud/calendar?branch=master)
 [![Dependency Status](https://www.versioneye.com/user/projects/57333497a0ca35004cf77c92/badge.svg?style=flat)](https://www.versioneye.com/user/projects/57333497a0ca35004cf77c92)
 
-**A calendar app for [ownCloud](https://owncloud.org).**  
+**A calendar app for open cloud platforms like [ownCloud](https://owncloud.org) and [Nextcloud](http://nextcloud.com).**
```
Comment (MariusBluem) [inline README.md]: "for ownCloud and Nextcloud" seems to be fine. No need for the "open cloud platforms like" stuff ;)

File: README.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 [![Code Coverage](https://scrutinizer-ci.com/g/owncloud/calendar/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/owncloud/calendar/?branch=master)
 [![Coverage Status](https://coveralls.io/repos/github/owncloud/calendar/badge.svg?branch=master)](https://coveralls.io/github/owncloud/calendar?branch=master)
 [![Dependency Status](https://www.versioneye.com/user/projects/57333497a0ca35004cf77c92/badge.svg?style=flat)](https://www.versioneye.com/user/projects/57333497a0ca35004cf77c92)
 
-**A calendar app for [ownCloud](https://owncloud.org).**  
+**A calendar app for open cloud platforms like [ownCloud](https://owncloud.org) and [Nextcloud](http://nextcloud.com).**  
 
-This is the new calendar app that has replaced the [old calendar app](https://github.com/owncloudarchive/calendar) with ownCloud 9.0, and is only the frontend for the new CalDAV-Backend which has moved into the [ownCloud Core](https://github.com/owncloud/core). 
+This is the new calendar app that has replaced the [old calendar app](https://github.com/owncloudarchive/calendar), and is only the frontend for the new integrated CalDAV-Backend.
```
Comment (MariusBluem) [inline README.md]: Integrated into what? - "for the new server-integrated CalDAV-Backend describes it better I think :wink:

File: appinfo/info.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
 	<dependencies>
 		<owncloud min-version="9.0" max-version="9.2" />
 	</dependencies>
-	<description>Calendar GUI for ownCloud's integrated CalDAV server</description>
+	<description>Calendar GUI for the integrated CalDAV server</description>
```
Comment (MariusBluem) [inline appinfo/info.xml]: Integrated into what? - "for the new server-integrated CalDAV-Backend describes it better I think :wink:

File: issue_template.md [hunk 4]
```diff
@@ -25,16 +25,16 @@ Tell us what happens instead
 
 **PHP version:**
 
-**ownCloud version:** (see ownCloud admin page)
+**ownCloud / Nextcloud version:** (see your admin page)
```
Comment (MariusBluem) [inline issue_template.md]: Just call it "server" :wink:

File: issue_template.md [hunk 5]
```diff
@@ -44,15 +44,15 @@ paste the results here.
...
+from within your installation folder
 ```
 
 **The content of config/config.php:**
 
 ```
 If you have access to your command line run e.g.:
 sudo -u www-data php occ config:list system
-from within your ownCloud installation folder
+from within your cloud installation folder
```
Comment (MariusBluem) [inline issue_template.md]: `server` is better then `cloud` I think :grin:

File: issue_template.md [hunk 6]
```diff
@@ -94,9 +94,9 @@ Eventually replace sensitive data as the name/IP-address of your LDAP server or
 Insert your webserver log here
 ```
 
-#### ownCloud log (data/owncloud.log)
+#### log file (data/owncloud.log)
```
Comment (MariusBluem) [inline issue_template.md]: Log instead of log ... Capital letters at the beginning :wink:

**Comment by MariusBluem** [comment]
See my comments @Bugsbane :wink:

**Comment by Bugsbane** [comment]
Ok, I like all those suggestions, @mar1u5 . Implemented.

**Comment by MariusBluem** [comment]
You have missed one :sweat_smile: ... :+1: for the fast changes.

**Comment by MariusBluem** [comment]
THX. Great Initiative :facepunch: ... @georgehrke

File: js/package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "calendar",
-  "description": "Calendar Application for ownCloud",
+  "description": "Integrated Calendar Application",
```
Comment (georgehrke) [inline js/package.json:3]: please change to `Calendar Application for ownCloud/Nextcloud`

**Comment by georgehrke** [comment]
Looks good except for the one I commented 👍

**Comment by Bugsbane** [comment]
> please change to Calendar Application for ownCloud/Nextcloud

The idea is to eliminate using "ownCloud/Nextcloud" wherever possible, to keep the wording as platform neutral as possible. To that end, we've already eliminated a number of the other places this was used with a better alternative. I mean, it's only ownCloud and Nextcloud at the moment, but who's to say what could happen (eg some other platform adds support for OC/NC apps or a project changes their name, or another fork or...) and we're back doing this all again.

Right now, the calendar app description is being displayed in the ownCloud and soon the Nextcloud app store. Saying that an app there is for ownCloud / Nextcloud is redundant. It's similar to having every app in the Google Play store state that it works on Android, or every app on iTunes state that it works on iOS. That's understood and assumed already, and adding it in just distracts from what is actually unique about the app.

If we use the description to describe what the app does, I'm sure people will understand by where it's hosted, what platform it's for.

**Comment by georgehrke** [comment]
The `js/package.json` is something only relevant to developers. Users will never ever see this file.

**Comment by MariusBluem** [comment]
> I mean, it's only ownCloud and Nextcloud at the moment, but who's to say what could happen (eg some other platform adds support for OC/NC apps or a project changes their name, or another fork or...) and we're back doing this all again.

I think you are better informed then me, if you think that there is any project planning to support apps coded for ownCloud/Nextcloud, or that anybody wants to rename their product ... Another fork?! Nah ... I think this is not going to happen ... There are no reasons currently 😄 @Bugsbane 

The `owncloud/calendar` project, and especially the maintainers (for example @georgehrke or @raghunayyar) are **not** going to make a neutral product which is independent itself, and works on nearly every platform and fork IMHO. This is working for ownCloud and Nextcloud. And the current plan is, to keep it compatible for both projects!

File: issue_template.md [hunk 8]
```diff
@@ -44,15 +44,15 @@ paste the results here.
...
+from within your installation folder
 ```
 
 **The content of config/config.php:**
 
 ```
 If you have access to your command line run e.g.:
 sudo -u www-data php occ config:list system
-from within your ownCloud installation folder
+from within your server installation folder
```
Comment (georgehrke) [inline issue_template.md]: please remove the word server so it's consistent with line 47 and line 74

Comment (MariusBluem) [inline issue_template.md]: I think we should call it here, and in every other line (47, 74, ...) `instance folder`😕 What do you think? @georgehrke

Comment (georgehrke) [inline issue_template.md]: Yes, like that :)
👍

Comment (Bugsbane) [inline issue_template.md]: > I think you are better informed then me, if you think that...

I have no idea what's going to happen in the future, which is why I'm uncomfortable with assuming. Even if nothing changes though "for ownCloud/Nextcloud" is still redundant information for an app on the ownCloud/Nextcloud app stores.

> I think we should call it here, and in every other line (47, 74, ...) instance folder

Makes sense. I'll update to add that.

**Comment by Bugsbane** [comment]
> I think you are better informed then me, if you think that...

I have no idea what's going to happen in the future, which is why I'm uncomfortable with assuming. Even if nothing changes though "for ownCloud/Nextcloud" is still redundant information for describing an app on the ownCloud/Nextcloud app stores.

> I think we should call it here, and in every other line (47, 74, ...) instance folder

Makes sense. I'll update to add that.

**Comment by MariusBluem** [comment]
:+1: from me ... Lets ask @georgehrke for a final review :wink:

**Comment by MariusBluem** [comment]
...and @DeepDiver1975

**Comment by raghunayyar** [comment]
:+1: from my end too. Thx for the initiative.

**Comment by raghunayyar** [comment]
can you squash the commits please? @georgehrke what do you think?

**Comment by georgehrke** [comment]
No need to squash, we can just use GitHub's squash and merge feature

File: issue_template.md [hunk 9]
```diff
@@ -71,7 +71,7 @@ Insert your config.php content here
 ```
 With access to your command line run e.g.:
 sudo -u www-data php occ ldap:show-config
-from within your ownCloud installation folder
+from within your installation folder
```
Comment (georgehrke) [inline issue_template.md]: this one needs to be changed to `instance installation folder` as well :)

**Comment by MariusBluem** [comment]
Can you change the repository description from 
`📅 Calendar app for ownCloud 9+` to `📅 Calendar app for ownCloud/Nextcloud 9+` ?

@georgehrke

**Comment by georgehrke** [comment]
Yes, will do

**Comment by georgehrke** [comment]
merged manually
https://github.com/owncloud/calendar/commit/667a861045d5a53bb32302a811a2c89b53b324dc

**Comment by Bugsbane** [comment]
Thanks for all your help with this @mar1u5 / @georgehrke

**Pull request closed without merging**